lumapps_api_client/lib.py

```diff
@@ -343,28 +374,27 @@ def get_matching_methods(self, method_parts):
         # find 'startswith' matches of the last part
         last = method_parts[-1]
         idx = len(method_parts) - 1
-        matches = [m for m in matches if len(m) >= idx and m[idx].startswith(last)]
+        matches = [m for m in matches
+                   if len(m) >= idx and m[idx].startswith(last)]
         if not matches:
             return "API method not found"
-        return (
-            "API method not found. Did you mean any of these?\n"
-            + self.get_method_descriptions(sorted(matches))
-        )
+        return ("API method not found. Did you mean any of these?\n" +
+                self.get_method_descriptions(sorted(matches)))
 
     def set_customer_user(self, email, customerId, instanceId=None):
         token = self.get_call("user", "getToken", email=email, customerId=customerId)
         self._user = {}
         if token and token.get("accessToken"):
+            TOKEN_VALIDITY_DURATION = 60 * 60
             self.token_expiration = (
                 datetime.now() + timedelta(seconds=TOKEN_VALIDITY_DURATION)
             ).isoformat()[:19]
             self.token = token.get("accessToken")
             self.email = email
             self.customerId = customerId
             self.instanceId = instanceId
-
+            import logging
```

Comment:
You are right! There should not be any Logging in this file because lib.py can be used outside of APP Engine context; Therefore logging may not exist. Removed in a new commit :)